tests/foreman/api/test_http_proxy.py

```diff
@@ -125,3 +128,69 @@ def test_positive_end_to_end(setup_http_proxy, module_target_sat, module_manifes
     )
     assert len(yum_repo['output']) >= 1
     assert 'quay/busybox' in yum_repo['output']
+
+
+@pytest.mark.upgrade
+@pytest.mark.rhel_ver_match('8')
+@pytest.mark.run_in_one_thread
+@pytest.mark.parametrize(
+    'setup_http_proxy',
+    [None, True, False],
+    indirect=True,
+    ids=['no_http_proxy', 'auth_http_proxy', 'unauth_http_proxy'],
+)
+@pytest.mark.tier3
+def test_positive_auto_attach(
+    setup_http_proxy, module_target_sat, rex_contenthost, function_entitlement_manifest_org
+):
+    """Attempt to auto attach a subscription to content host
+
+    :id: cce888b5-e023-4ee2-bffe-efa9260224ee
+
+    :customerscenario: true
+
+    :expectedresults: host successfully subscribed, subscription
+        repository enabled, and repository package installed.
+
+    :Assignee: jpathan
+
+    :BZ: 2046337
+
+    :parametrized: yes
+
+    :CaseLevel: System
+    """
+    org = function_entitlement_manifest_org
+    with manifests.clone() as manifest:
+        upload_manifest(org.id, manifest.content)
```

Comment:
```suggestion
```
Uploading the manifest is performed by the `function_entitlement_manifest_org` fixture, so it's not necessary to do so in the test body.